packages/forma-36-react-components/src/components/CopyButton/README.mdx

```diff
@@ -0,0 +1,34 @@
+import { CopyButton } from './CopyButton';
+
+# CopyButton
+
+The CopyButton is a styled button that copies text into the user's clipboard.
```

Comment:
@denkristoffer I'm not sure how we use this component right now, but maybe it's worth mentioning how it might be used?